package.json

```diff
@@ -32,6 +32,7 @@
     "core-js": "^2.4.1",
```

Comment:
This pull request is for changing the Peace Corps logo. If we need to have flexibility in versions we need to create a new issue for it and then proceed.